js-intermediate.js

```diff
@@ -45,41 +53,83 @@ var originalArray1 = [1, 2, 3, 4, 5, 6, 7]
 var originalArray2 = ["a", "t", "l", "e", "d"]
 // Expected output: ["d", "e", "l", "t", "a"]
 
+let reverse = (arr) =>{
+  return arr.reverse('')
+}
 
-
+console.log(reverse(originalArray1))
 // 5. Write the code that would make this test pass.
 
-describe("multTwo", () => {
-    test ("returns an array with all the numbers multiplied by two", () => {
-        expect(multTwo([3, 4, 5])).toEqual([6, 8, 10])
-        expect(multTwo([23, -9, 0])).toEqual([46, -18, 0])
-        expect(multTwo([4.5, -4.5, 12])).toEqual([9, -9, 24])
-    })
-})
-
+// describe("multTwo", () => {
+//     test ("returns an array with all the numbers multiplied by two", () => {
+//         expect(multTwo([3, 4, 5])).toEqual([6, 8, 10])
+//         expect(multTwo([23, -9, 0])).toEqual([46, -18, 0])
+//         expect(multTwo([4.5, -4.5, 12])).toEqual([9, -9, 24])
+//     })
+// })
+
+let multTwo = (arr) =>{
+  let newArr = arr.map(value => value * 2)
+return newArr
+}
 
+// console.log(multTwo(arr))
 
 // 6. Write a function called letterCounter that logs the number of times the letter "l" appears in ourString.
 
 var ourString = "Hello Learn Students!"
 // Expected output: 3
 
+let letterCounter = (str) =>{
+  let arr = str.toLowerCase().split('')
+  letterCount = 0
+  let letter = "l"
+  for(let i=0; i<arr.length; i++){
+    if(arr[i].includes(letter))
+    letterCount += 1
+    }return letterCount
+  }
 
+console.log(letterCounter(ourString))
 
 // 7. Create a function that takes in a string of a single word and returns the middle letter of the word. If the word is an even number of letters, return the two middle letters.
 
 var middleLetters1 = "hello"
 // Expected output: “l”
-var middleLetters2 = "lamas"
+var middleLetters2 = "llamas"
 // Expected output: “am”
 
-
-
+let middleLetter = (str) =>{
```

Comment:
Take a look at this again. The expected output for even inputs is to take the middle two letters.